ui/packages/catalog-ui-search/src/main/webapp/component/ingest-details/ingest-details.view.js

```diff
@@ -72,6 +72,7 @@ module.exports = Marionette.LayoutView.extend({
     this.render()
     this.onBeforeShow()
     this.resetDropzone()
+    this.uploadBatchModel.clear()
```

Comment:
```
    resetDropzone() {
       this.dropzone.options.autoProcessQueue = false
       this.dropzone.removeAllFiles(true)
    }
```

Might be worth grouping this into the resetDropzone function for future developers to have clarity.  What I'm suggesting above would have the same effect as this line.